src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -4372,19 +4372,19 @@ trait Types
   def elementExtractOption(container: Symbol, tp: Type): Option[Type] = {
     elementExtract(container, tp) match {
       case NoType => None
-      case tp => Some(tp)
+      case oktype => Some(oktype)
```

Comment:
could allow `x.f(y, z) match { case x | y | z  => }` on the same criteria as `tp.dealias`. Not going to fix this now, maybe later. Oh but this code needs to pass the lint, so I guess I will the source for now.

Reply:
💡